casbin/config/config.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#ifdef CASBIN_EXPORTS
+#define CONFIG_API __declspec(dllexport)
+#else
+#define CONFIG_API __declspec(dllimport)
+#endif
+
+#include<string>
+#include<mutex>
+#include<map>
+#include<sstream>
+#include<vector>
+
+
+using namespace std;
+
+extern const string DEFAULT_SECTION;
+extern const string DEFAULT_COMMENT;
+extern const string DEFAULT_COMMENT_SEM;
+extern const string DEFAULT_MULTI_LINE_SEPARATOR;
+
+
+
+class CONFIG_API Config {
+public:
+	map<string, map<string, string>> data;
+	mutex m;

```

Comment:
No need to add mutex here, because a config is read only once, won't have data races.